docs/C-Summary/CSS/index.css

```diff
@@ -169,3 +172,45 @@ main{
     linear-gradient(#000 0 0);
   -webkit-mask-composite: destination-out;
 }
+
+@media only screen and (max-width: 375px){
+  .attribution{
+    display: none;
+  }
+  main{
+    position: static;
+    width: auto;
+    height: auto;
+    margin: 0;
+    padding: 0;
+    border-radius: 0;
+    box-shadow: none;
+    transform: translate(0%, 0%);
+    -ms-transform: translate(0%, 0%);
+  }
+  .left-section{
+    position: static;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: 0;
+    border-radius: 0px 0px 25px 25px;
+  }
+  .right-section{
+    position: static;
```

Comment:
Just curious: why do you choose static?

Reply:
I chose static because .right-section was already defined with position absolute but that won't work with the responsive version and with `static` it removes the utility of the positioning i gave it earlier:`top, left, bottom, right`. Although i realized thanks to this i can remove the positioning i was giving it in `@media` because it doesn't work with static positioning.